test/integration/interrupt_resume_test.rb

```diff
@@ -104,4 +104,172 @@ def test_interrupt_resume_when_table_has_completed
 
     assert_test_table_is_identical
   end
+
+  def test_interrupt_resume_inline_verifier_with_datawriter
+    datawriter = new_source_datawriter
+    ghostferry = new_ghostferry(MINIMAL_GHOSTFERRY, config: { verifier_type: "Inline" })
+
+    start_datawriter_with_ghostferry(datawriter, ghostferry)
+
+    batches_written = 0
+    ghostferry.on_status(Ghostferry::Status::AFTER_ROW_COPY) do
+      batches_written += 1
+      if batches_written >= 2
```

Comment:
After every row copy, integrationferry sends a HTTP request to the ruby integration test `AFTER_ROW_COPY` and the DataIterator is paused until the ruby code here returns and returns a 200 back to the integrationferry. Since we're only copying a single table, there's no concurrency. 

Is this what you mean?